<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>902</width>
    <height>761</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,1">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QPushButton" name="configureInstruments">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Edit configuration</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="Line" name="line">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="runButton">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Run experiment</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="Line" name="line_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="recordButton">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Record data</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Record duration in seconds (0=forever):</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="recordDuration">
          <property name="maximum">
           <double>10000000.000000000000000</double>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="autoIncrementSample">
          <property name="text">
           <string>Move to next sample after recording</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="Line" name="line_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Samples</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QListWidget" name="samplesList">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="selectionMode">
             <enum>QAbstractItemView::SingleSelection</enum>
            </property>
            <property name="selectionRectVisible">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <widget class="QPushButton" name="addSample">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>Add sample</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="removeSample">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>Remove sample</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="moveUp">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>Move up</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="moveDown">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>Move down</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="verticalSpacer">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTabWidget" name="instrumentTabs">
        <property name="currentIndex">
         <number>-1</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>902</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionNewConfig"/>
    <addaction name="actionOpenConfig"/>
    <addaction name="actionSaveConfig"/>
    <addaction name="actionSaveConfigAs"/>
    <addaction name="actionReloadConfig"/>
    <addaction name="actionClose"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpenConfig">
   <property name="text">
    <string>&amp;Open config</string>
   </property>
   <property name="iconText">
    <string>Open Config</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSaveConfig">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&amp;Save config</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionReloadConfig">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&amp;Reload config</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>Close</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionInstruments">
   <property name="text">
    <string>&amp;Instruments...</string>
   </property>
  </action>
  <action name="actionasdf">
   <property name="text">
    <string>asdf</string>
   </property>
  </action>
  <action name="actionSaveConfigAs">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Save config as...</string>
   </property>
  </action>
  <action name="actionNewConfig">
   <property name="text">
    <string>&amp;New config</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
